bootstrap/bungeecord/src/main/java/org/geysermc/geyser/platform/bungeecord/GeyserBungeePingPassthrough.java

```diff
@@ -59,7 +61,17 @@ public GeyserPingInfo getPingInformation(InetSocketAddress inetSocketAddress) {
                 future.complete(event);
             }
         }));
-        ProxyPingEvent event = future.join();
+
+        ProxyPingEvent event;
+
+        try {
+            event = future.get(100, TimeUnit.MILLISECONDS);
+        } catch (Throwable cause) {
+            proxyServer.getLogger().log(Level.SEVERE, "Failed to get ping information for " + inetSocketAddress, cause);
```

Comment:
Please use the Geyser logger here - you can access it via `GeyserImpl.getInstance().getLogger()`.